action.yml

```diff
@@ -53,16 +53,22 @@ runs:
         env:
           VERSION: ${{ inputs.version }}
         run: |
+          CPU=$(uname -m)
```

Comment:
It looks like you could use the [runner.arch context variable](https://docs.github.com/en/actions/learn-github-actions/contexts#runner-context) here to make this just a tad cleaner?

Reply:
Done. Also added X86-32 and ARM32, because GitHub runner.arch lists them as possible.